pkg/apis/kops/dockerconfig.go

```diff
@@ -18,27 +18,22 @@ package kops
 
 // DockerConfig is the configuration for docker
 type DockerConfig struct {
-	Bridge   *string `json:"bridge,omitempty" flag:"bridge"`
-	LogLevel *string `json:"logLevel,omitempty" flag:"log-level"`
-	IPTables *bool   `json:"ipTables,omitempty" flag:"iptables"`
-	IPMasq   *bool   `json:"ipMasq,omitempty" flag:"ip-masq"`
-
-	LogDriver string   `json:"logDriver,omitempty" flag:"log-driver"`
-	LogOpt    []string `json:"logOpt,omitempty" flag:"log-opt,repeat"`
-
+	Bridge        *string  `json:"bridge,omitempty" flag:"bridge"`
+	LogLevel      *string  `json:"logLevel,omitempty" flag:"log-level"`
+	IPTables      *bool    `json:"ipTables,omitempty" flag:"iptables"`
+	IPMasq        *bool    `json:"ipMasq,omitempty" flag:"ip-masq"`
+	LogDriver     string   `json:"logDriver,omitempty" flag:"log-driver"`
+	LogOpt        []string `json:"logOpt,omitempty" flag:"log-opt,repeat"`
+	DefaultUlimit []string `json:"defaultUlimit,omitempty" flag:"default-ulimit,repeat"`
```

Comment:
We need to start adding more validation methods. We can do a follow up PR if you like. You mind adding some code level comments?